src/download.ts

```diff
@@ -159,54 +115,44 @@ function download(url: URL, filePath: string) {
  * @param cliPath
  * @returns
  */
-async function ensureBinaryRuns(cliPath: string): Promise<string> {
-  const signResult = await new SimpleExecutor().execute({ command: cliPath, args: ['version'], ignoreError: true }, {});
-
-  if (signResult.exitCode === 0) {
-    outputChannel.appendLine('The latest CodeScene CLI is in place, we are ready to go!');
-    return cliPath;
-  } else {
-    throw new Error(`Error invoking the downloaded CLI binary: ${signResult.stderr}`);
-  }
+async function ensureBinaryRuns(cliPath: string) {
```

Comment:
Maybe not needed when we also run isExpectedCliVersion when validating the installed binary

Reply:
Yes, but we need to ensure it runs after being downloaded as well. `ensureBinaryRuns` was originally intended for catching the rare "missing instruction set" error so it was simplest to just reuse that.
I remember thinking about just running `isExpectedCliVersion` again, maybe I'll do that and just throw a general invocation error if it returns false.